zetaclient/orchestrator/bootstrap.go

```diff
@@ -481,6 +478,20 @@ func syncObserverMap(
 	return added, removed, nil
 }
 
+func btcDatabaseFileName(chain chains.Chain) string {
+	// btcDatabaseFilename is the Bitcoin database file name now used in mainnet and testnet3
+	// so we keep using it here for backward compatibility
+	const btcDatabaseFilename = "btc_chain_client"
```

Comment:
Should this remain a package variable rather than being declared in the function? `const btcLegacyDatabaseFilename` ?

I guess the benefit of declaring it here is that other can't accidentally reference it

Reply:
yeah. other code don't need it